nautobot/extras/tests/test_customfields.py

```diff
@@ -1200,6 +1200,14 @@ def test_filter_text(self):
 
     def test_filter_date(self):
         self.assertEqual(self.filterset({"cf_cf5": "2016-06-26"}, self.queryset).qs.count(), 1)
+        self.assertEqual(self.filterset({"cf_cf5__lte": "2016-06-28"}, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset({"cf_cf5__lte": "2016-06-27"}, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset({"cf_cf5__lte": "2016-06-26"}, self.queryset).qs.count(), 1)
+        self.assertEqual(self.filterset({"cf_cf5__lte": "2016-06-25"}, self.queryset).qs.count(), 0)
+        self.assertEqual(self.filterset({"cf_cf5__gte": "2016-06-25"}, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset({"cf_cf5__gte": "2016-06-26"}, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset({"cf_cf5__gte": "2016-06-27"}, self.queryset).qs.count(), 1)
+        self.assertEqual(self.filterset({"cf_cf5__gte": "2016-06-28"}, self.queryset).qs.count(), 0)
```

Comment:
I feel like we also add a compound filter test using `__gte` and `__lte` to assert that is also functioning as expected? Perhaps one that is in the range (`lte 2016-06-28, gt 2016-06-25`) and one out?